v2/apiserver/schemas/job-status.json

```diff
@@ -36,7 +36,7 @@
 		"phase": {
 			"type": "string",
 			"description": "The job's phase",
-			"enum": [ "ABORTED", "CANCELED", "FAILED", "PENDING", "RUNNING", "SCHEDULING_FAILED", "STARTING", "SUCCEEDED", "UNKNOWN" ]
+			"enum": [ "ABORTED", "CANCELED", "FAILED", "PENDING", "RUNNING", "SCHEDULING_FAILED", "STARTING", "SUCCEEDED", "TIMED_OUT", "UNKNOWN" ]
```

Comment:
I think it wasn't an accident that `TIMED_OUT` was missing here.

More on this in another comment. Will link back to here...